$NetBSD: patch-ak,v 1.1 2009/12/29 18:12:44 asau Exp $

Make MAIN__ weak in order to allow non-Fortran programs to use the library.

--- main.c.orig	2008-04-06 06:42:46.000000000 +0400
+++ main.c	2009-12-29 18:18:21.000000000 +0300
@@ -49,6 +49,7 @@
 #define Int /* int */
 #else
 extern void f_init(void), sig_die(const char*, int);
+#pragma weak MAIN__
 extern int MAIN__(void);
 #define Int int
 #endif
